app/views/browse/_replica-set-details.html

```diff
@@ -5,6 +5,8 @@
         deployment-config="deploymentConfig"
         pods="podsForDeployment"
         hpa="autoscalers"
+        quotas="quotas"
+        cluster-quotas="clusterQuotas"
```

Comment:
yeah i changed all the templates before i decided not to add the quota polling in the RS controller